ecs-cli/modules/cli/compose/project/project_parseV3_test.go

```diff
@@ -110,6 +116,12 @@ func TestParseV3WithOneFile(t *testing.T) {
 			IpAddress: aws.String("10.0.0.0"),
 		},
 	}
+	mysqlCon.HealthCheck = &ecs.HealthCheck{
+		Command:  aws.StringSlice([]string{"CMD-SHELL", "curl -f http://theecscliisawesome.com || exit 1"}),
```

Comment:
it sure is! ;) Although, why is "CMD-SHELL" included here when it's not in the docker-compose file?

Reply:
From my design doc and the compose docs:
```
# As above, but wrapped in /bin/sh. Both forms below are equivalent.
test: ["CMD-SHELL", "curl -f http://localhost || exit 1"]
test: curl -f https://localhost || exit 1
```
When test is specified as a string, "CMD-SHELL" gets added.

Reply:
Since I've had to explain this more than once in this PR, I'm gonna add a comment above that line to make explain it. So that there's no confusion when future maintainers look at this test.

Reply:
I was just going to suggest that 👍